lib/trento_web/openapi/schema/database.ex

```diff
@@ -0,0 +1,80 @@
+defmodule TrentoWeb.OpenApi.Schema.Database do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  alias TrentoWeb.OpenApi.Schema.{ResourceHealth, Tags}
+
+  defmodule DatabaseInstance do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "DatabaseInstance",
+      description: "A discovered HANA Database Instance on the target infrastructure",
+      type: :object,
+      properties: %{
+        sap_system_id: %Schema{type: :integer, description: "SAP System ID", format: :uuid},
+        sid: %Schema{type: :string, description: "SID"},
+        tenant: %Schema{type: :string, description: "Tenant"},
+        instance_number: %Schema{type: :string, description: "Instance Number"},
```

Comment:
The instance number is a number usually, but as they add the initial `0` most of the times, I guess having as string is still valid

Reply:
Yeah, true.